docs/polaris/hardware-overview/machine-overview.md

```diff
@@ -37,17 +37,18 @@ Note 2: 8 memory channels rated at 204.8 GiB/s
 |              |               | mlx5_0 | SYS  | SYS  | SYS  | PHB  | X      | SYS    |
 |              |               | mlx5_1 | SYS  | PHB  | SYS  | SYS  | SYS    | X      |
 
-### Legend:
+!!! info "Legend"
```

Comment:
At least just in the markdown render, this just renders as `!!! info "Legend"` -- Or is this something in CSS or HTML that will render correctly on the website?

Reply:
viewing the doc source in GitHub or in your local Markdown previewer is not a reliable representation of what the docs will look like once deployed. I recommend doing https://github.com/argonne-lcf/user-guides?tab=readme-ov-file#preview-the-docs-locally-and-test-for-errors for anything but the most minor of changes to this repo

Reply:
<img width="733" alt="image" src="https://github.com/user-attachments/assets/bb738afc-f58b-4892-a23b-e3388be3f198" />

Reply:
Oh, thanks for the pointer! It looks nice indeed.